airbyte-integrations/connectors/destination-redshift/src/main/java/io/airbyte/integrations/destination/redshift/operations/RedshiftSqlOperations.java

```diff
@@ -112,7 +114,19 @@ protected void insertRecordsInternalV2(final JdbcDatabase database,
         // > default
         for (final List<PartialAirbyteMessage> batch : Iterables.partition(records, 10_000)) {
           LOGGER.info("Prepared batch size: {}, {}, {}", batch.size(), schemaName, tableName);
-          final DSLContext create = using(connection, SQLDialect.POSTGRES);
+          final DSLContext create = using(
+              connection,
+              SQLDialect.POSTGRES,
+              // Force inlined params.
+              // jooq normally tries to intelligently use bind params when possible.
+              // This would cause queries with many params to use inline params,
+              // but small queries would use bind params.
+              // In turn, that would force us to intelligently escape string values,
+              // since we need to escape inlined strings
+              // but need to not escape bound strings.
+              // Instead, we force jooq to always inline params,
+              // and always call escapeStringLiteral() on the string values.
+              new Settings().withStatementType(StatementType.STATIC_STATEMENT));
```

Comment:
I was confused reading their docs.  This will switch from prepared statement to inlined statement. 
There is another setting `.withParamType(ParamType.INLINED)` which i think probably is what we are intending to do ? maybe the test will verify the expectation.

Reply:
nvm. just saw your resolved comment.